### ISSUE TITLE
Update cleos output for get account

### ISSUE THREAD

**Issue by bytemaster** [EOSIO/eos#3666]
```
easypayx@easypayxs-MacBook-Pro:~/eos-dawn-5-22/eos/build/programs/cleos $ ./cleos get account eosio
privileged: true
permissions: 
     owner     1:    1 EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV
        active     1:    1 EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV
memory: 
     quota:        -1 bytes  used:      2.66 Kb   

net bandwidth: (averaged over 3 days)
     used:                -1 bytes
     available:           -1 bytes
     limit:               -1 bytes

cpu bandwidth: (averaged over 3 days)
     used:                -1 us   
     available:           -1 us   
     limit:               -1 us  

```

Instead of "-1" it should say "unlimited"

We shouldn't even display "privileged" unless it is true... most accounts it is false.

**Comment by dhaneshvb** [comment]
https://github.com/EOSIO/eos/pull/3819/files

**Comment by heifner** [comment]
Resolved by #3819
